react/Breadcrumb.js

```diff
@@ -40,51 +41,43 @@ class Breadcrumb extends Component {
   }
 
   render() {
-    const {
-      slug,
-      productQuery: { product, loading },
-    } = this.props
+    const { term, categories } = this.props
 
-    if (loading || !product || !product.categories) {
+    if (!categories) {
       return null
     }
 
-    const categoriesList = this.getCategories(product.categories)
+    const categoriesList = this.getCategories(categories)
     return (
       <div className={CSS_CLASSES.BREADCRUMB}>
         <Link className={CSS_CLASSES.LINK} page="store">
-          {this.props.intl.formatMessage({ id: 'breadcrumb.home' })}
+          <HomeIcon />
```

Comment:
Add a `<span>` with a token in `className`

Reply:
👍